typebackend/views.py

```diff
@@ -39,6 +46,33 @@ def post(self,request):
         else:
             return Response({'success':False,'error':serializer.errors},status=status.HTTP_400_BAD_REQUEST)
 
+class RegisterForGoogleUsers(APIView):
+    def post(self,request):
+        token=request.data['token']
+        username=request.data['username']
+        id_info=id_token.verify_oauth2_token(token,requests.Request(),config('CLIENT_ID'))
+        email=id_info['email']
```

Comment:
Why are we not verifying `id_info`? If it's invalid, what does it return?
Pretty sure if it's invalid, id_info won't contain "email" and this line will throw an error which will cause server to crash.